### PR TITLE
chore: update DEVELOPER file

### PR DESCRIPTION
**Pull request by Foxandxss**
I continue bikeshedding a bit, but I updated this file to give instructions on how to use the tooling and well, also added more sections from `angular` DEVELOPER file.

### CODE REVIEW

File: DEVELOPER.md [hunk 1]
```diff
@@ -1,4 +1,110 @@
...
+1. Login to your GitHub account or create one by following the instructions given
+   [here](https://github.com/signup/free).
+2. [Fork](http://help.github.com/forking) the [main ng-bootstrap
+   repository](https://github.com/ng-bootstrap/core).
+3. Clone your fork of the ng-bootstrap's core repository and define an `upstream` remote pointing back to
+   the ng-bootstrap's core repository that you forked in the first place.
+
+```shell
+# Clone your GitHub repository:
+git clone git@github.com:<github username>/core.git
```
Comment (wesleycho) [inline DEVELOPER.md]: This should maybe be `git clone git@github.com:<github username>/core.git ng-bootstrap` so people have it in a better named directory.

Comment (Foxandxss) [inline DEVELOPER.md]: I can see that.

**Comment by Foxandxss** [comment]
Fixed that thingy.

**Comment by wesleycho** [comment]
LGTM

**Pull request closed without merging**